src/Commands/Concerns/InteractsWithIO.php

```diff
@@ -29,17 +28,15 @@ trait InteractsWithIO
     ];
 
     /**
-     * Write a string as raw output.
+     * Gets the log format, if any.
      *
-     * @param  string  $string
-     * @return void
+     * @return string|null
      */
-    public function raw($string)
+    public function logFormat()
```

Comment:
I don't think we can remove a method on a stable release here?

Reply:
I can add it back - but honestly, this is internal API, people should not really try extend octane commands...

Reply:
@nunomaduro I also very much doubt anyone's doing that but I've seen stranger things happening in the past 😅

Reply:
Since this PR does not remove the calls to `raw` in `StartSwooleCommand` for example, this for sure cannot be safe or am I missing something?

Reply:
`@internal` tag if your friend. ;)